core/app/models/spree/order_updater.rb

```diff
@@ -183,6 +183,7 @@ def update_item_promotions
         Spree::Config.promotion_chooser_class.new(promotion_adjustments).update
 
         item.promo_total = promotion_adjustments.select(&:eligible?).sum(&:amount)
+        item.save
```

Comment:
This value isn't checked anywhere, so if necessary this should use `save!`. We need a spec to show why this is necessary. After removing this everything still passes.